airbyte-integrations/connectors/destination-snowflake/src/test-integration/kotlin/io/airbyte/integrations/destination/snowflake/typing_deduping/AbstractSnowflakeTypingDedupingTest.kt

```diff
@@ -336,6 +336,47 @@ abstract class AbstractSnowflakeTypingDedupingTest : BaseTypingDedupingTest() {
         }
     }
 
+    @Test
+    fun testAirbyteMetaAndGenerationIdMigrationForOverwrite() {
```

Comment:
can you add another test case that runs two old syncs? 3.9.1 -> 3.10.0 -> dev

just to verify that we can support the wrongly-migrated people (i.e. destination state says that we already did the migration)